### ISSUE TITLE
[WPF] No support for "lighter" weight in visualizer?

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#2888]
Unclear if it's a hostconfig, implementation, or platform issue. This card:

```json
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.2",
  "body": [
	{
	  "type": "RichTextBlock",
	  "inlines": [
		{
		  "type": "TextRun",
		  "text": "lighter",
          "weight": "lighter"
		},
		{
		  "type": "TextRun",
		  "text": "default",
          "weight": "default"
		},
		{
		  "type": "TextRun",
		  "text": "bolder",
          "weight": "bolder"
		}
	  ]
    }
  ]
}
```

yields this output in UWP (correct):
![image](https://user-images.githubusercontent.com/16614499/57557867-792b0e00-7330-11e9-9dbd-56c90cbb7393.png)

but yields this output in WPF:
![image](https://user-images.githubusercontent.com/16614499/57557880-85af6680-7330-11e9-991c-24875306d9ac.png)

seems like `lighter` isn't working?

**Comment by shalinijoshi19** [comment]
@almedina-ms / @paulcam206 does this need to be fixed for 1.2 ? It looks like a tooling bug yes?

**Comment by shalinijoshi19** [comment]
@almedina-ms is already gearing this up for 1.2 PR. PUlling it back in.
